### PR TITLE
Add sceFiber

### PR DESCRIPTION
**Pull request by frangarcj**

### CODE REVIEW

File: include/psp2/fiber.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#ifndef _PSP2_FIBER_H
+#define _PSP2_FIBER_H
+
+#include <psp2/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SCE_FIBER_ERROR_NULL    		0x80590001
```
Comment (xyzz) [inline include/psp2/fiber.h]: can you make these enums?

**Comment by frangarcj** [comment]
Fixed

File: include/psp2/fiber.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/**
+ * \file
+ * \brief Header file which defines SSL variables and functions
```
Comment (DaveeFTW) [inline include/psp2/fiber.h]: >SSL variables and function

This needs corrected.

**Comment by frangarcj** [comment]
Fixed

**Pull request merged**